packages/core/src/scripts/cli.ts

```diff
@@ -89,7 +89,7 @@ export async function cli (cwd: string, argv: string[]) {
   }
 
   if (command === 'start') {
-    return start(cwd, defaultFlags(flags, { ui: true, withMigrations: false }))
+    return start(cwd, defaultFlags(flags, { server: true, ui: true, withMigrations: false }))
```

Comment:
`start --with-migrations --no-server` might be redundant by comparison to `keystone prisma migrate deploy`, but, I think `start --no-server` being ignored is worse